### ISSUE TITLE
Hexagonal computer keyboard layout

### ISSUE THREAD

**Issue by codedot** [cifkao/tonnetz-viz#3]
Feature request: in addition to piano-like key-mapping for computer keyboard, provide Tonnetz-based mapping similar to [Tonnetz-based MIDI controllers](http://c-thru-music.com/cgi/index.cgi).

**Comment by cifkao** [comment]
@codedot Great idea! Do you have a specific layout in mind? I would use the Sonome/Harmonic Table layout rotated by 90° so as to match the Riemannian Tonnetz. (It has to be rotated somehow anyway, otherwise it won't fit the keyboard.)

**Comment by codedot** [comment]
One way to go with a keyboard layout that looks sane to me is as follows:
```
1_C    2_G    ...   0_D#   -_A#   =_F
q_E    w_B    ...   p_G    [_D    ]_A
a_G#   s_D#   ...   ;_B    '_F#
z_C    x_G    ...   /_D#
```
Maybe, it also makes sense to allow transposition by fifths using the left and right arrows and by major thirds using the up and down arrows.

**Comment by cifkao** [comment]
Yes, that's exactly what I had in mind.

**Comment by cifkao** [comment]
Resolved by 7035ec1.
